### PR TITLE
little typo fix

### PR DESCRIPTION
**Pull request by acfnews**
Hessu, please have a look.
I believe all warn's are now correct 'do syslog's'.

### CODE REVIEW

**Comment by hessu** [comment]
This patch still breaks indentation for almost all of the changes. Please use the same style of indentation - the original file has 1 TAB character for each level of indentation. If you look at the diff on github you'll see each changed line jump much deeper than the original (I guess you used 8 spaces per level). Thanks!

File: rip44d [hunk 1]
```diff
@@ -437,7 +456,7 @@ my $next_expire = time() + $expire_interval;
 
 # Main loop: receive broadcasts, check that they're from the correct
 # address and port, and pass them on to processing
-warn "entering main loop, waiting for RIPv2 datagrams\n" if ($verbose);
+do_syslog("entering main loop, waiting for RIPv2 datagrams") if ( $verbose >= 0 );
```
Comment (hessu) [inline rip44d]: Umm, this should be if ($verbose) - it's _always_ going to be >= 0.

File: rip44d [hunk 2]
```diff
@@ -450,14 +469,14 @@ while (1) {
 	my $addr_s = inet_ntoa($peer_addr);
 	
 	if ($addr_s ne '44.0.0.1' || $peer_port ne 520) {
-		warn "$me: ignored packet from $addr_s: $peer_port: " . length($msg) . "\n";
+                do_syslog("ignored packet from $addr_s: $peer_port: " . length($msg) ) if ( $verbose >= 1 );
```
Comment (hessu) [inline rip44d]: Please do not change the logic of ($verbose). We'll always want to log suspicious errors such as this one even without ($verbose).

File: rip44d [hunk 3]
```diff
@@ -249,37 +266,37 @@ sub validate_route($$$$$)
...
 	my $prefix_len = mask2prefix($e_netmask);
 	if ($prefix_len < 0) {
-		warn "invalid netmask: $e_netmask_s\n" if ($verbose);
+                do_syslog("invalid netmask: $e_netmask_s") if ($verbose);
 		return (0, 'invalid netmask');
 	}
 	
 	if ($prefix_len < $minimum_prefix_len) {
-		warn "$e_net_s/$e_netmask_s => $e_nexthop_s blocked, prefix too short\n";
+                do_syslog("$e_net_s/$e_netmask_s => $e_nexthop_s blocked, prefix too short") if ($verbose);
```
Comment (hessu) [inline rip44d]: This case is an error, should be logged even without ($verbose). I would not like to change the logic of the verbose switch even with syslog.

**Comment by acfnews** [comment]
new version created

**Pull request closed without merging**